src/msg/Message.h

```diff
@@ -304,6 +304,7 @@ class Message : public RefCountedObject {
 				   &Message::dispatch_q>> Queue;
 
   ceph::mono_time queue_start;
+  bool is_prepared = false;
```

Comment:
@rzarzynski okay, I was thinking of it as a field that can signify when a message is ready to be sent. Do you mean I should move it to ProtocolV1/V2 or find a different way to do this?

Reply:
@rzarzynski ping

Reply:
Actually `ProtocolV2` doesn't need this new field as it already uses the `is_prepared` local variable. This is nice because a `ProtocolV2`-specific thing is kept within the `ProtocolV2` boundaries.

I think for `ProtocolV1` it's harder as the knowledge needs be shared across two methods of the class. I wonder whether we could modify the V1's `out_q` queue (`_get_next_outgoing()` is getting the `m` pointer from there) to convey the `is_prepared` boolean. It already stores `pair<ceph::buffer::list, Message *>` per single message; maybe this could become a `tuple` or, preferably, `struct` to name the fields.